docs/tutorials/annealed_sinkhorn.ipynb

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
<p>in the left plot I think the transport solver has not converged, it should have values across all columns, but none are visible in most columns</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>